sklearn/utils/estimator_checks.py

```diff
@@ -778,7 +778,7 @@ def _set_checking_parameters(estimator):
 
     if name == "QuantileRegressor":
```

Comment:
I think you can completely remove the `if name == "QuantileRegressor"` block. Basically the previous code was needed when the default depended on the scipy version but now it is always `higgs` and there is no need to set it ...

Reply:
Removed.